tools/fix_auth/fix_auth.rb

```diff
@@ -39,11 +39,11 @@ def models
     end
 
     def generate_password
-      ManageIQ::Password.generate_symmetric("#{cert_dir}/v2_key")
+      ManageIQ::Password.generate_symmetric("#{cert_dir}/v2_key", ENV["APPLIANCE"] ? "manageiq" : nil)
```

Comment:
Since we are extending the API (and will thus need to cut a `v1.1.0`, I'm wondering if we should use a keyword param here instead, for readability...So, I'm picturing something like:

```suggestion
      ManageIQ::Password.generate_symmetric("#{cert_dir}/v2_key", groupname: ENV["APPLIANCE"] ? "manageiq" : nil)
```

Reply:
that looks much nicer